pkg/occlient/occlient_test.go

```diff
@@ -3221,3 +3221,49 @@ func TestUpdateDCToSupervisor(t *testing.T) {
 		})
 	}
 }
+
+func TestIsVolumeAnEmptyDir(t *testing.T) {
+	type args struct {
+		VolumeName string
+		dc         appsv1.DeploymentConfig
+	}
+	tests := []struct {
+		name         string
+		args         args
+		wantEmptyDir bool
+		actions      int
+	}{
+		{
+			name: "Case 1 - Check that it is an emptyDir",
+			args: args{
+				VolumeName: supervisordVolumeName,
+				dc:         *fakeDeploymentConfig("foo", "bar"),
+			},
+			wantEmptyDir: true,
+			actions:      1,
+		},
+		{
+			name: "Case 2 - Check a non-existent volume",
+			args: args{
+				VolumeName: "foobar",
+				dc:         *fakeDeploymentConfig("foo", "bar"),
+			},
+			wantEmptyDir: false,
+			actions:      1,
+		},
```

Comment:
there should probably be one more test case for volume that exists, but it is something else then emptyDir